btyacc/PKGBUILD

```diff
@@ -1,25 +1,25 @@
 
 _realname=byacc
 pkgname=btyacc
-pkgver=20200910
+pkgver=20210109
 pkgrel=1
 pkgdesc="btyacc - an LALR(1) parser generator with support for backtracking"
 arch=('i686' 'x86_64')
 url="https://invisible-island.net/byacc"
 license=('Public Domain')
 groups=('base-devel')
 
-source=(${_realname}-${pkgver}.tar.gz::https://invisible-mirror.net/archives/byacc/byacc-${pkgver}.tgz)
-sha256sums=('0a5906073aeaf23ddc20aaac0ea61cb5ccc18572870b113375dec4ffe85ecf30')
+source=(${_realname}-${pkgver}.tar.gz::https://github.com/ThomasDickey/byacc-snapshots/archive/t${pkgver}.tar.gz)
```

Comment:
What made you select/trust that source?

Reply:
It's the package maintainer's github repo. Otherwise it is likely that CI cannot compile when downloading from the former website.

Reply:
ok, I'm missing any kind of link between the page and the github account. but ok I guess.